app/tests/spec/lib/user-agent.js

```diff
@@ -100,6 +100,50 @@ define(function (require, exports, module) {
       });
     });
 
+    describe('isDesktop', () => {
+      it('returns `true` if on a desktop device', () => {
```

Comment:
For unknown user agents, this would also be displayed? Should it we worded as `if not on mobile`?